statsmodels/stats/tests/test_diagnostic.py

```diff
@@ -1705,13 +1705,23 @@ def test_ljungbox_auto_lag_selection():
     data = sunspots.load_pandas().data["SUNACTIVITY"]
     res = AutoReg(data, 4, old_names=False).fit()
     resid = res.resid
-    res1 = smsdia.acorr_ljungbox(resid)
-    res2 = smsdia.acorr_ljungbox(resid, model_df=4)
+    res1 = smsdia.acorr_ljungbox(resid, auto_lag=True)
+    res2 = smsdia.acorr_ljungbox(resid, model_df=4, auto_lag=True)
     assert_allclose(res1.iloc[:, 0], res2.iloc[:, 0])
+    #TODO: compare selected lags with Stata/ R to confirm that corect auto_lag is selected
+    assert res1.shape[0] >= 1
+    assert res2.shape[0] >= 1
     assert np.all(np.isnan(res2.iloc[:4, 1]))
     assert np.all(res2.iloc[4:, 1] <= res1.iloc[4:, 1])
 
 
+def test_ljungbox_auto_lag_whitenoise():
+    data = np.random.rand(1000) #white noise process
+    res = smsdia.acorr_ljungbox(data, auto_lag=True)
+    #TODO: compare selected lags with Stata/ R to confirm that corect auto_lag is selected
```

Comment:
Can you add a space after #

Reply:
Hi! I've made the PEP fixes, and added the space after the comment marker. There is an R package ["vrtest"](https://rdrr.io/cran/vrtest/man/Auto.Q.html) that supposedly implements the Escanciano & Lobato (2009) automatic lag selection procedure. However, the function only returns one LB statistic and one p-value, presumably for the optimal lag. The LB statistic seems way too small to be accurate --- indeed, smaller than the LB stat for even the first lag according to the `statsmodels` procedure. So not sure what's happening there. I'm a total R noob, so perhaps someone else can help out also?

Reply:
Can you use randn instead of rand? Also, can you add `reset_randomstate` to the function, e.g. `test_ljungbox_auto_lag_whitenoise(reset_randomstate)`?